src/dbnode/persist/fs/seek_manager.go

```diff
@@ -376,7 +376,9 @@ func (m *seekerManager) newOpenSeeker(
 	shard uint32,
 	blockStart time.Time,
 ) (DataFileSetSeeker, error) {
-	exists, err := DataFileSetExistsAt(m.filePathPrefix, m.namespace, shard, blockStart)
+	// TODO(juchan): get the actual volume here.
```

Comment:
Do we need to do this for this PR merges or are we tracking this required change in an issue/elsewhere where it won't get forgotten?

Reply:
Not needed for merging (since with cold writes disabled the volume number can only ever be 0).

My other P.R that you reviewed will address this issue and allow Justin to rebase off of that or justin can just file an issue and do in a quick follow up. @justinjc

Reply:
I've opened https://github.com/m3db/m3/issues/1747 to track.